### PR TITLE
Fixes describe command keyword output

### PR DESCRIPTION
**Pull request by atallahhezbor**

### CODE REVIEW

File: geomesa-tools/bin/download-data.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env bash
+
+type=$1
```
Comment (jahhulbert-ccri) [inline geomesa-tools/bin/download-data.sh]: what happens if ```$1``` is empty

File: geomesa-tools/bin/download-data.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  wget "http://research.microsoft.com/pubs/152883/User_guide_T-drive.pdf" -P $GEOMESA_HOME/data/tdrive
+
+else
+
+  if [ -n "$type" ]; then
+    PREFIX="Unknown data type '$type'."
+  else
+    PREFIX="Please enter a data type."
+  fi
+  echo "$PREFIX Available types: gdelt, geolife, osm-gpx, tdrive"
```
Comment (jahhulbert-ccri) [inline geomesa-tools/bin/download-data.sh]: prefer this syntax in bash for var substitution when possible ```"${PREFIX}"```

File: geomesa-tools/bin/download-data.sh (hunk 1 above)
Comment (elahrvivaz) [inline geomesa-tools/bin/download-data.sh]: it's handled in the else on line 60

File: geomesa-tools/bin/geomesa [hunk 3]
```diff
@@ -146,10 +146,15 @@ function findJars() {
...
 
+function findSftDirs() {
+    DIRS=$(find $GEOMESA_CONF_DIR/sfts -mindepth 1 -type d -exec printf {}: \;)
+    echo ${DIRS%:}
+}
 # Start constructing GEOMESA_CP (classpath)
 # include geomesa first so that the correct log4j.properties is picked up
 GEOMESA_CP="${GEOMESA_CONF_DIR}:$(findJars $GEOMESA_LIB/common)"
 GEOMESA_CP="${GEOMESA_CP}:${ACCUMULO_CONF_DIR}:${HADOOP_CONF_DIR}"
+GEOMESA_CP="${GEOMESA_CP}:${GEOMESA_CONF_DIR}:$(findSftDirs)"
```
Comment (jahhulbert-ccri) [inline geomesa-tools/bin/geomesa]: does this work OK with no dirs?

File: geomesa-tools/conf/sfts/geonames/reference.conf [hunk 4]
```diff
@@ -0,0 +1,64 @@
+geomesa =
```
Comment (jahhulbert-ccri) [inline geomesa-tools/conf/sfts/geonames/reference.conf]: picky - move { up a line multiple times in this file to match the others

**Comment by jahhulbert-ccri** [comment]
i'd say the only thing to think about is if we ever wanted to reserve ```$GEOMESA_HOME/data``` for something else. I can't think of what it would be so that's probl moot.

File: geomesa-tools/bin/geomesa (hunk 3 above)
Comment (atallahhezbor) [inline geomesa-tools/bin/geomesa]: Yup. They simple don't get added to the classpath and don't show up in ```geomesa env``` or ```geomesa classpath```.

**Pull request closed without merging**